src/main/java/mate/academy/bookstore/repository/order/OrderRepository.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy.bookstore.repository.order;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.bookstore.model.Order;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
```

Comment:
```suggestion
```

Reply:
not mandatory

Reply:
well, doesnt that improve code readability?